tasks/main.yaml

```diff
@@ -14,6 +14,16 @@
       - podman
     state: present
 
+- name: Create an empty mounts.conf to override podman default mounts
+  # See also: man containers-mounts.conf (shipped by containers-common)
+  copy:
+    content: ""
+    dest: /etc/containers/mounts.conf
+    force: false
```

Comment:
maybe I should use `force: true` instead?

Reply:
I believe we want `force: true` in this case to make sure that result is always empty file.

Reply:
Makes sense, I've updated it `true`.